ambry-api/src/test/java/com.github.ambry/account/AccountContainerTest.java

```diff
@@ -41,20 +38,25 @@
 public class AccountContainerTest {
   private static final Random random = new Random();
   private static final int CONTAINER_COUNT = 10;
+  private static final short LATEST_CONTAINER_JSON_VERSION = Container.JSON_VERSION_2;
 
   // Reference Account fields
   private short refAccountId;
   private String refAccountName;
   private AccountStatus refAccountStatus;
-  private JSONObject accountJsonLike;
+  private JSONObject refAccountJson;
 
   // Reference Container fields
   private List<Short> refContainerIds;
   private List<String> refContainerNames;
   private List<String> refContainerDescriptions;
   private List<ContainerStatus> refContainerStatuses;
-  private List<Boolean> refContainerPrivacyValues;
-  private List<JSONObject> containerJsonLikeList;
+  private List<Boolean> refContainerEncryptionValues;
```

Comment:
I was wondering whether we really need all of these lists of reference fields. In a lot of cases we could just compare using the getters for the `refContainers`. I suppose in some cases you might be testing Container against itself, but these would just be getter methods.

Reply:
I agree - again follow up PR maybe. Create an issue with all this "fluff" removal?